server/src/test/java/org/elasticsearch/common/settings/SettingTests.java

```diff
@@ -277,14 +277,14 @@ public void testDefault() {
 
         // It gets more complicated when there are two settings objects....
         Settings hasFallback = Settings.builder().put("foo.bar", "o").build();
-        Setting<String> fallsback =
+        Setting<String> fallsBack =
```

Comment:
If we're going to change this one, I think it should just be `fallback`

Reply:
I normally would have picked `fallback`, the only reason I didn't was that I was wondering if the code was trying to speak about the action of falling back as opposed to the actual fallback.

I'm quite happy to switch (and will when I get to the code side, I think that'll come after the API bits get split out).